Project.toml

```diff
@@ -1,24 +1,31 @@
 name = "GlobalDiffEq"
 uuid = "1d72d19b-84cc-4cb7-a099-7cbdb9ccc67c"
 authors = ["Chris Rackauckas <accounts@chrisrackauckas.com> and contributors"]
-version = "0.1.1"
+version = "0.2.0"
```

Comment:
Is this breaking? If not, adding features to a pre-version-1 packages should only bump the patch release.

Reply:
Sorry, I am new to versioning. I wanted to mark this change as breaking because it has new dependencies such as Distributions. However, I would be happy to change it to 0.1.2.